ui/components/app/user-preferenced-currency-display/user-preferenced-currency-display.component.js

```diff
@@ -25,19 +29,22 @@ export default function UserPreferencedCurrencyDisplay({
   });
   const prefixComponent = useMemo(() => {
     return (
-      currency === EtherDenomination.ETH &&
-      showEthLogo && (
-        <i
-          className="fab fa-ethereum"
-          style={{
-            color: 'var(--color-icon-default)',
-            fontSize: ethLogoHeight,
-          }}
+      showEthLogo &&
+      currency === nativeCurrency && (
+        <AvatarNetwork
```

Comment:
Using AvatarNetwork here because after checking font-awesome it seemed like there is icon only for ethereum;
Also I'm showing the icon when currency=== native currency and not just when currency === ETH

Reply:
This is good for now, though we may want to make this more friendly by changing to have a startIcon prop that simply renders the icon if its provided. We could then have a new component that just renders this component with the AvatarNetwork as the icon prop, isolating the additional selectors and reducing the impact of mocks needing to be updated in tests.